base/tests/tool-git.yaml

```diff
@@ -0,0 +1,16 @@
+- desc: git should be installed
+  command: [which, dsadsada]
```

Comment:
Undo this.

Reply:
Bug: tests on the base image is not working or configured correctly. This test case should have failed.